docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -303,29 +303,27 @@ You can only edit the settings of Data Assets created in GX Cloud.
 
 3. Click **Save**.
 
-## Secure your GX API Data Source connection strings
+## Data Source credential management
 
-When you use the GX API and not GX Cloud to connect to Data Sources, you can obfuscate sensitive Data Source credentials in your connection string as an additional security measure.
+To connect to your database in GX Cloud, there are two methods for managing credentials:
 
-1. Store your credential value as an environment variable by entering `export ENV_VAR_NAME=env_var_value` in the terminal or adding the command to your `~/.bashrc` or `~/.zshrc` file. For example:
+1. **Direct input**: You can input credentials directly into GX Cloud. These credentials are securely encrypted at rest and in transit, stored in our database, and accessible to the GX Agent and any API user.
 
-    ```bash title="Terminal input"
-    export GX_CLOUD_SNOWFLAKE_PASSWORD=<password-string>
-    ```
-    Prefix environment variable names with `GX_CLOUD_`.
+2. **Environment variable substitution**: To enhance security, you can use environment variables to manage sensitive connection parameters or strings. For example, instead of directly including your database password in configuration settings, you can use a variable reference like `${MY_DATABASE_PASSWORD}`:
```

Comment:
```suggestion
2. **Environment variable substitution**: To enhance security, you can use environment variables to manage sensitive connection parameters or strings. For example, instead of directly including your database password in configuration settings, you can use a variable reference like `${MY_DATABASE_PASSWORD}`. When using environment variable substitution, your password is not stored or transmitted to GX Cloud.
```